lib/node_modules/@stdlib/math/base/special/hyp2f1/README.md

```diff
@@ -0,0 +1,105 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2018 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# hyp2f1
+
+> Evaluate Gaussian hypergeometric function
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+The Gaussian hypergeometric function is defined as
+
+<!-- <equation class="equation" label="eq:gaussian_hypergeometric_function" align="center" raw="hyp2f1(a,b,c,z) = \sum_{n=0}^{\infty} \frac{(a)_n (b)_n}{(c)_n} \frac{z^n}{n!}" alt="Equation for Gaussian hypergeometric function."> -->
+
+<div class="equation" align="center" data-raw-text="hyp2f1(a,b,c,z) = \sum_{n=0}^{\infty} \frac{(a)_n (b)_n}{(c)_n} \frac{z^n}{n!}" data-equation="eq:gaussian_hypergeometric_function">
```

Comment:
Cool.  Thanks for looking at it so far.  There are definitely some implementation issues to work out because I have a lot of failing tests.  I probably just forgot to transfer something over from the C code or something.  Thanks for the heads up on the testing command too!